src/Altinn.Notifications.Core/Services/OrderRequestService.cs

```diff
@@ -33,7 +33,7 @@ public OrderRequestService(IOrderRepository repository, IGuidService guid, IDate
     }
 
     /// <inheritdoc/>
-    public async Task<(NotificationOrder? Order, ServiceError? Error)> RegisterNotificationOrder(NotificationOrderRequest orderRequest)
+    public async Task<Result<NotificationOrder, ServiceError>> RegisterNotificationOrder(NotificationOrderRequest orderRequest)
```

Comment:
Do we need ServiceError here, it will never be returned?

Reply:
Saving order to db could fail, but we don't catch the exception for now. So either we catch exceptions from the repository layer and return service error or drop the service error.

Reply:
Drop is good :)